static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -103,7 +104,7 @@
     display: flex;
     flex-direction: column;
 
-    @include mq($from: mobileLandscape) {
+    @include mq($from: 580px) {
```

Comment:
Why does the layout need to break at this very specific width? Perhaps this is related to what @gustavpursche has already mentioned about using `background-image`. Can we aim for something from the [`$mq-breakpoints`](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/_vars.scss#L51-L60)?